src/commands/utility/changeStatus.js

```diff
@@ -25,7 +26,7 @@ module.exports = {
 
       if (!channel.isThread()) {
         return await interaction.reply({
-          content: 'Sorry, this is not a thread!',
+          content: translateLanguage('changeStatus.notAThread'),
```

Comment:
maybe some errors are reusable along the application, other errors are specific of certain feature, so those that are reusable could be on `"errors.nameOfError"`, and those that are specific could be in `"feature.errors.nameOfError"`

so in this case, what do you think name it `"errors.notAThread"`, since it sounds like an error that can be reused in other places of the app?